### DISCUSSION TITLE
Disable automatic tagging

### DISCUSSION THREAD

**Discussion by dimejo** [cupcakearmy/autorestic#149] [Q&A]
Is it possible to disable automatic tagging (`ar:cron` and `ar:location:NAME`) of snapshots? This messes with my forget policy because I'm using `--group-by host,tags`. I currently need to remove those tags with before-hooks which isn't nice.

**Comment by cupcakearmy** [comment]
I see. Mhh kinda tricky. the forget and restore commands use the tags to identify the snapshots. Will need to think about a good solution.
